script/utils/update-redirects-data.ts

```diff
@@ -42,6 +42,10 @@ const updateRedirectsData = async(currentBook: BookWithOSWebData, newBook: BookW
     ) {
       redirects.push(formatSection(section));
       countNewRedirections++;
+    } else if (!slug) {
+      // tslint:disable-next-line: no-console
+      console.error(`page with ID ${section.id} not found in new book`);
```

Comment:
lets put a comment in here like "remove this to enable legitimately removing pages from books only once uuids are guaranteed to be consistent"